acai-ios/MenuTabView.swift

```diff
@@ -0,0 +1,153 @@
+//
+//  MenuTabView.swift
+//  acai-ios
+//
+//  Created by Drew Dunne on 3/28/19.
+//  Copyright © 2019 Cornell AppDev. All rights reserved.
+//
+//  Adapted from Jack Thompson's EpisodeFilterView from Ceres
+
+import UIKit
+import SnapKit
+
+class MenuTabView: UIControl {
+
+    // MARK: - Subview Variables
+    private var stackView: UIStackView!
+    var underline: UIView!
+    private var buttons: [UIButton] = []
+    private var items: [String]!
+
+    // MARK: - Control Data
+    var selectedSegmentIndex: Int = 0 {
+        didSet {
+            remakeUnderlineConstraints()
+        }
+    }
+
+    // MARK: - Properties
+    var underlineHeight: CGFloat = 2.5 {
+        didSet {
+            remakeUnderlineConstraints()
+        }
+    }
+
+    var underlineOffset: CGFloat = 3.0 {
+        didSet {
+            remakeUnderlineConstraints()
+        }
+    }
+
+    var titleAttributes: [NSAttributedString.Key: Any] = [
+        .font: UIFont.avenirNextMedium.withSize(14),
+        .foregroundColor: UIColor.white
+        ] {
+        didSet {
+            setButtonTitleAttributes()
+        }
+    }
+
+    var contentInset: CGFloat = 32.0 {
```

Comment:
do these vars need to be public?

Reply:
Yeah these are settable to customize the look in case we want to reuse